Description: Build with the "read-only relocation" hardening linker flag
 Pass "-Wl,-z,relro" to the linker. This fixes lintian warning 
 "hardening-no-relro".
Forwarded: http://trac.wildfiregames.com/ticket/1416
Author: Vincent Cheng <vcheng@debian.org>
Last-Update: 2013-09-05

--- a/build/premake/premake4.lua
+++ b/build/premake/premake4.lua
@@ -281,7 +281,7 @@
 				end
 
 				if os.is("linux") or os.is("bsd") then
-					linkoptions { "-Wl,--no-undefined", "-Wl,--as-needed" }
+					linkoptions { "-Wl,--no-undefined", "-Wl,--as-needed", "-Wl,-z,relro" }
 				end
 
 				if arch == "x86" then
